pkg/config/nodetreemodel/config.go

```diff
@@ -285,6 +324,25 @@ func (c *ntmConfig) leafAtPath(key string) LeafNode {
 	return missingLeaf
 }
 
+// GetNode returns a Node for the given key
+func (c *ntmConfig) GetNode(key string) (Node, error) {
```

Comment:
Do we want this to be public for now ?

Reply:
Yes, I meant to leave a note, but this can be used by UnmarshalKey so that it doesn't build new nodes right away.